lib/postgresql.js

```diff
@@ -522,6 +522,9 @@ PostgreSQL.prototype.buildExpression = function(columnName, operator,
       const regexOperator = operatorValue.ignoreCase ? ' ~* ?' : ' ~ ?';
       return new ParameterizedSQL(columnName + regexOperator,
         [operatorValue.source]);
+    case '@>':
```

Comment:
I find the operator name `@>` as very cryptic. Have you considered using a more descriptive name like `contains`? 

Having said that, maybe the operator `@>` is something that PostgreSQL users are familiar with, and a new name like `contains` would just add learning overhead?

Let's discuss.

Reply:
Another aspect to consider: this operator can be used in REST API clients too, e.g. to call 

```http
GET https://somehost/api/posts?filter={"where":{"category":{"@>":["AA"]}}}
```

From the point of view of REST API consumers (e.g. front-end developers), I think a descriptive operator name like `contains` is much better.

```http
GET https://somehost/api/posts?filter={"where":{"category":{"contains":["AA"]}}}
```

Reply:
Done renamed operator from @> to contains